README.md

```diff
@@ -9,8 +9,8 @@ jsk openni tools
 
 ### Create catkin workspace
 ```
-$ mkdir-p catkin_ws/ws_jsk_openni_kinect/src
-$ cd catkin_ws/ws_jsk_openni_kinect
+$ mkdir-p ~/catkin_ws/ws_jsk_openni_kinect/src
```

Comment:
Can you create PR for this 

--
◉ Kei Okada

2016/05/30 11:33、iory <notifications@github.com> のメッセージ:

> In README.md:
> 
> > @@ -9,8 +9,8 @@ jsk openni tools
> >  
> >  ### Create catkin workspace
> >  ```
> > -$ mkdir-p catkin_ws/ws_jsk_openni_kinect/src
> > -$ cd catkin_ws/ws_jsk_openni_kinect
> > +$ mkdir-p ~/catkin_ws/ws_jsk_openni_kinect/src
> typo
> mkdir -p ~/catkin_ws/ws_jsk_openni_kinect/src
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>